app/controllers/employee/operations_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Employee::OperationsController < EmployeeController
+
+  def new
+    @operation = Operation.new
+    @parcel = Parcel.new
+  end
+
+  def create
+    @operation = Operation.new(operation_params)
+    @parcel = Parcel.new(parcel_params)  # serves as a container for form data
+    if @operation.place.present? && @operation.save
+      redirect_to new_employee_operation_path, notice: t('.operation_created_succesfully')
+    else
+      flash[:alert] = t('.no_place_provided') if @operation.place.blank?
+      render 'new'
+    end
+  end
+
+  private
+
+  def operation_params
+    parcel = Parcel.find_by(parcel_params)
+    params.require(:operation).permit(:kind, :place, :additional_info).merge(user: current_employee, parcel: parcel)
```

Comment:
Lekko to nieczytelne i generalnie mam wątpia do używania metod typu `merge` na paramsach. Już prędzej ten merge zrobiłbym przy przekazaniu do konstruktora.